styles/About.module.css

```diff
@@ -0,0 +1,148 @@
+.container {
+    width: 100%;
+    height: auto;
+    background-color: white;
+}
+
+.main {
+    display: flex;
+    flex-direction: column;
+    padding: 50px 50px 15px 50px;
+    width: 100%;
+    height: auto;
+}
+
+.about {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height:auto;
+    position: relative;
+}
+
+.about__description {
```

Comment:
@LRode could not figure out how to place the Moon photo between this box and the box's shadow ( box-shadow: 15px 15px #5CA2D5;)